src/config/mod.rs

```diff
@@ -1988,6 +2003,40 @@ impl Dimensions {
     }
 }
 
+/// Window Geometry
+#[serde(default)]
+#[derive(Default, Debug, Copy, Clone, Deserialize, PartialEq, Eq)]
+pub struct Geometry {
+    /// Window x-position
+    #[serde(deserialize_with = "failure_default")]
+    x: i32,
+
+    /// Window y-position
+    #[serde(deserialize_with = "failure_default")]
+    y: i32,
+}
+
+impl Geometry {
+    pub fn new(x: i32, y: i32) -> Self {
+        Geometry {
+            x,
+            y,
+        }
+    }
+
+    /// Get x-pos
+    #[inline]
+    pub fn x(self) -> i32 {
+        self.x
+    }
+
+    /// Get y-pos
+    #[inline]
+    pub fn y(self) -> i32 {
+        self.y
+    }
```

Comment:
I think it just making these x and y fields public is a better choice than having getters here.